src/classes/CW_CampaignSitesTriggerHandler.cls

```diff
@@ -0,0 +1,109 @@
+global class CW_CampaignSitesTriggerHandler {
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.